.gitlab-ci.yml

```diff
@@ -12,7 +12,7 @@ test:
   tags:
     - gcloud
   script:
-    - git clone --depth 1 https://github.com/nearprotocol/nearcore.git nearcore
+    - git clone --depth 1 --branch staging https://github.com/nearprotocol/nearcore.git nearcore
```

Comment:
I think it's good idea to do both, similar to how @janedegtiareva did for `near-shell`:
https://github.com/nearprotocol/near-shell/pull/138

Reply:
I think master is not going to work, since this staging change is breaking the results.